.github/workflows/release.yml

```
@@ -0,0 +1,171 @@
+          git tag -a "${{ steps.version.outputs.VERSION }}" -m "Release ${{ steps.version.outputs.VERSION }}"
+
+          # Push tag
+          git push origin "${{ steps.version.outputs.VERSION }}"
+
+      - name: 📦 Build distribution
+        run: |
+          python -m build --wheel --sdist
+
+      - name: 📋 Generate release notes from PR
+        id: release_notes
+        run: |
```

actionlint (1.7.10)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)